### PR TITLE
TranslatorText: fixed "score" field type

### PR DESCRIPTION
**Pull request by filariow**
The go client for TranslatorText returned the following error:

translatortext.TranslatorClient#Translate: Failure responding to request: StatusCode=200 -- Original Error: Error occurred unmarshalling JSON - Error = 'json: cannot unmarshal number 1.0 into Go struct field TranslateResultAllItemDetectedLanguage.detectedLanguage.score of type int32' JSON = '[{"detectedLanguage":{"language":"en","score":1.0},"translations":[{"text":"text","to":"en"}]}]'

The score value is supposed to be a floating point number not an integer.
Hope this is the right fix for the problem.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 1]
```diff
@@ -894,7 +894,7 @@
                 "type": "string"
               },
               "score": {
-                "type": "integer"
+                "type": "float"
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:897]: This is not the right way to document a float
https://swagger.io/docs/specification/data-models/data-types/

**Review by lmazuel — CHANGES_REQUESTED** [review]
Incorrect Swagger, see https://swagger.io/docs/specification/data-models/data-types/ for reference on how to do that

**Comment by FrancescoIlario** [comment]
> Incorrect Swagger, see https://swagger.io/docs/specification/data-models/data-types/ for reference on how to do that

You're right, sorry for that.
I updated to the (i hope) correct format.
Is here ok float or is it preferred double?

Thank you for the review

**Comment by lmazuel** [comment]
Hi @FrancescoIlario , the right choice between float or double would be to the service team (pinging @yangyuan ). I know Swagger, I don't know the service :). Thanks!

**Pull request closed without merging**